rs/crypto/secp256k1/src/lib.rs

```diff
@@ -620,6 +620,20 @@ impl PrivateKey {
         Ok(tweaked_key.sign_message_with_bip340(message, rng))
     }
 
+    /// Sign a message with BIP340 Schnorr with Taproot derivation, without using an external RNG
+    ///
+    /// The aux_rand parameter for BIP340 is just to re-randomize the signature,
+    /// and may prevent certain forms of fault attack. It it is otherwise not necessary
+    /// for security.
```

Comment:
```suggestion
    /// Sign a message with BIP341 Schnorr with Taproot derivation, without using an external RNG
    ///
    /// The aux_rand parameter for BIP341 is just to re-randomize the signature,
    /// and may prevent certain forms of fault attack. It it is otherwise not necessary
    /// for security.
```

Reply:
nit: how about `sign_message_with_bip341_fixed_rng`?

Reply:
`_no_rng` is used for the similar `bip340` function seems better to be consistent one way or another

Reply:
Both `no_rng` and `fixed_rng` are fine with me, but indeed we should be consistent.

Reply:
`fixed_rng` looks to me more precise here because we actually do use an RNG in the function internally, but `no_rng` also makes sense because we don't pass an RNG, so I'm ok with both